package.json

```diff
@@ -48,7 +48,8 @@
   "scripts": {
     "commit": "git-cz",
     "prettier": "prettier **/*.{js,jsx,ts,tsx} scripts/**/*.js --write --ignore-path .gitignore",
-    "commitmsg": "validate-commit-msg"
+    "commitmsg": "validate-commit-msg",
+    "start": "MODE=local node ./lib/index.js"
```

Comment:
Is there a reason for this command? As opposed to the regular `npx create-contentful-app`?

Reply:
It's mainly a utility script to not forget to add the `MODE=local` whenever you develop locally. It was just the first idea off the top of my head, any other suggestions would be appreciated

Reply:
Maybe `npm run dev`?

Reply:
Pushed another commit with this change